### PR TITLE
Edge/cie 4179 update install takes more time

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Download the Cumulocity IoT Edge version 10.15 from the [Software AG Empower por
 
 Starting with Cumulocity IoT Edge Release 10.15, accessing the Edge appliance remotely from the Cumulocity IoT tenant using the Telnet protocol has been removed. Software AG recommends you to use the SSH protocol instead.
 
+### Delay in installation and update
```
Comment (bbyreddy) [inline content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md:15]: - [ ] Is the heading right by calling it as a "Delay"? We may have to find better wording.

Comment (sharath-jk) [reply content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md]: Updated the heading to "Installation and update processes take more time"

**Review by bbyreddy — APPROVED** [review]

File: content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ Download the Cumulocity IoT Edge version 10.15 from the [Software AG Empower por
 
 Starting with Cumulocity IoT Edge Release 10.15, accessing the Edge appliance remotely from the Cumulocity IoT tenant using the Telnet protocol has been removed. Software AG recommends you to use the SSH protocol instead.
 
+### Installation and update processes take more time
```
Comment (rssag) [inline content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md:15]:
```suggestion
### Installation and update processes take longer
```

File: content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md [hunk 3]
```diff
@@ -12,6 +12,10 @@ Download the Cumulocity IoT Edge version 10.15 from the [Software AG Empower por
...
+
+Installation and Update processes perform security hardening of the operating system and other components, due to which these processes take longer (approximately 50% higher) than the earlier versions.
```
Comment (rssag) [inline content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md:17]:
```suggestion
The installation and update processes perform security hardening of the operating system and other components.
As a result, these processes take approximately 50% longer than in earlier versions.
```

**Review by rssag — APPROVED** [review]

**Comment by rssag** [comment]
to which releases does this apply?

**Comment by sharath-jk** [comment]
@rssag - this applies to Edge 10.15 release.

**Pull request merged**